### PR TITLE
Publish packages

### PR DESCRIPTION
**Pull request by nichonien**

### CODE REVIEW

**Comment by nichonien** [comment]
@Timtech4u Would it be possible to add repository secrets for the `publish` script for npm release before we merge this? (C.C @jrhender )

**Comment by nichonien** [comment]
> @nichonien updated your PR, can you confirm that this is fine now?

@Timtech4u Thanks, looks good to me. Waiting for review from @jrhender.

File: packages/chains/package.json
Comment (jrhender) [inline packages/chains/package.json:3]: @nichonien The first part of the package name (prefixed with the `@`, until the `/`) is the scope of the name. https://docs.npmjs.com/cli/v9/using-npm/scope
So for packages published by EW, we should use the `@energyweb` scope (as we used for ew-credentials packages https://github.com/energywebfoundation/ew-credentials#description , for example), in my opinion. Perhaps a package name like `@energyweb/wagmi-chains` makes sense.
How do you see this?

File: .github/workflows/publish-package.yaml
Comment (jrhender) [inline .github/workflows/publish-package.yaml:22]: Better to use node v18 unless there is a reason for v16

File: .github/workflows/verify.yml
Comment (jrhender) [inline .github/workflows/verify.yml:16]: Does this not need to be set? Is it okay to remove without other changes?

File: .github/workflows/publish-package.yaml
Comment (jrhender) [inline .github/workflows/publish-package.yaml:1]: Was this file copied/inspired from a workflow elsewhere?

**Review by jrhender — CHANGES_REQUESTED** [review]
Thanks for the PR @nichonien . Added some comments. My main concern that the package name might have an incorrect scope.
Not crucial to add at this point, but in general it would be good to have a PR description that describes the purpose of the PR and notes any points to be aware of.

File: packages/chains/package.json
Comment (nichonien) [reply packages/chains/package.json:3]: Hmm, I agree with you. `@energyweb/wagmi-chains` makes more sense.

File: .github/workflows/verify.yml
Comment (nichonien) [reply .github/workflows/verify.yml:16]: @jrhender IMO we don't need any token to clone the repo, might be something that devs from `wagmi` have missed. This was corrected by @Timtech4u.

File: .github/workflows/publish-package.yaml
Comment (nichonien) [reply .github/workflows/publish-package.yaml:1]: We have similar publish workflow in `ew-credential` and `ew-did-registry` - https://github.com/energywebfoundation/ew-credentials/blob/develop/.github/workflows/deploy.yml. The tests and builds run missing here as there is no need.

File: packages/connectors/package.json
Comment (jrhender) [inline packages/connectors/package.json:2]: @nichonien what do you think of adding `wagmi` here as well?
```suggestion
  "name": "@energyweb/wagmi-connectors",
```

**Review by jrhender — APPROVED** [review]

**Pull request merged**
